pkg/env/node_scan_v4.go

```diff
@@ -1,10 +1,16 @@
 package env
 
 var (
-	// NodeScanningV4HostPath sets the path where the R/O host node filesystem is mounted to the container
+	// NodeIndexEnabled defines whether Compliance will actually run indexing code
+	NodeIndexEnabled = RegisterBooleanSetting("ROX_NODE_INDEX_ENABLED", false)
+
+	// NodeIndexHostPath sets the path where the R/O host node filesystem is mounted to the container
 	// that should be scanned by Scanners NodeIndexer
-	NodeScanningV4HostPath = RegisterSetting("ROX_NODE_SCANNING_V4_HOST_PATH", WithDefault("/host"))
+	NodeIndexHostPath = RegisterSetting("ROX_NODE_INDEX_HOST_PATH", WithDefault("/host"))
+
+	// NodeIndexContainerAPI Defines the API endpoint for the RepositoryScanner to reach out to
+	NodeIndexContainerAPI = RegisterSetting("ROX_NODE_INDEX_CONTAINER_API", WithDefault("https://catalog.redhat.com/api/containers/"))
 
-	// NodeScanningV4Enabled defines whether Compliance will actually run scanning code
-	NodeScanningV4Enabled = RegisterBooleanSetting("ROX_NODE_SCANNING_V4_ENABLED", false)
+	// NodeIndexMappingURL Defines the endpoint for the RepositoryScanner to download mapping information from
+	NodeIndexMappingURL = RegisterSetting("ROX_NODE_INDEX_MAPPING_URL", WithDefault("https://access.redhat.com/security/data/metrics/repository-to-cpe.json"))
```

Comment:
That would be a great idea, yes.
I changed the default URL for repo 2 cpe, but didn't find a central endpoint for the container API.
You mentioned you've got that disabled by default in Scanner currently, correct? Does this endpoint in central exist at all?

Reply:
I think we need to be careful to make sure these variables are set when we install the secured cluster because otherwise if the secured cluster is installed in a different cluster from central, this `https://central.stackrox.svc/api/extensions/scannerdefinitions?file=repo2cpe` is not going to be reachable.

I'm not sure if it should've being done here #12608 or if there's a planned follow up but I thought it would be a good idea to bring it up.

Reply:
> You mentioned you've got that disabled by default in Scanner currently, correct? Does this endpoint in central exist at all?

The consensus was: Yes it is disabled by default (see snippet below), however I do not see it disabled for `rhel_containerscanner` - worth re-confirming that we're not 'accidently' calling that API.
 
And no central does not currently proxy requests to that API

https://github.com/stackrox/stackrox/blob/1a03474942c3a09ccfe1d3064d0a7f23ae6138a9/scanner/indexer/indexer.go#L217-L232

Reply:
> I think we need to be careful to make sure these variables are set when we install the secured cluster because otherwise if the secured cluster is installed in a different cluster from central, this `https://central.stackrox.svc/api/extensions/scannerdefinitions?file=repo2cpe` is not going to be reachable.
> 
> I'm not sure if it should've being done here #12608 or if there's a planned follow up but I thought it would be a good idea to bring it up.

Agreed, I'm under the impression the current setting of the Central URL is temporary, though, and we should probably replace this is Sensor (and ensure Sensor will allow this traffic)